windows-installer/idt-win-installer.ps1

```diff
@@ -286,43 +263,20 @@ function install_plugins {
 }
 
 #------------------------------------------------------------------------------
-#-- Create "idt" script to act as shortcut to "bx dev"
+#-- Create "ic" script to act as shortcut to "ibmcloud"
 function env_setup() {
-  Write-Output "Creating 'idt' script to act as shortcut to 'ibmcloud dev' command..."
-  $idt_batch = @"
+  $ic_command = get-command ic -erroraction 'silentlycontinue'
+  if ( $ic_command ) {
+    # the command is set, use its existing location
+  } else {
+    Write-Output "Creating 'ic' script to act as shortcut to 'ibmcloud' command..."
+    $ic_batch = @"
 @ECHO OFF
-REM #-----------------------------------------------------------
```

Comment:
We should leave a comment header in the bat file, but just now say that it is an alias for `ibmcloud`.